hardware/wireless-chipsets/en.md

```diff
@@ -46,6 +46,7 @@ This list should not suggest that *only*- such devices listed below are compatib
 - Intel Ultimate N WiFi Link 5300
 - Intel WiFi Link 5100
 - Intel Wireless 8265 / 8275
+- Intel Corporation Wireless 7265
```

Comment:
Should be above "Intel Wireless 8265 / 8275" and just called "Intel Wireless 7265"